### DISCUSSION TITLE
Fixed: Moving the screen from one screen to another or changing the window from maximized let the ap crash

### DISCUSSION THREAD

**Discussion by arachnidus** [phcode-dev/phoenix#1918] [General]
Error Message: 'assertion failed: flush_paint_messages(None, &subclass_input.event_loop_runner)'
OS: Windows 11 23H2, three screens
![Screenshot 2024-10-28 112159](https://github.com/user-attachments/assets/b721ed9f-86b7-46ab-b912-18af6d6ac83e)

**Comment by abose** [comment]
Thanks for reporting the issue. Can you send the error report by pressing yes in the dialog for me to take a look.

**Reply by arachnidus** [reply]
This was already done before I started the discussion

**Comment by arachnidus** [comment]
This problem still exist, also with the most recent build. I keep using Brackets. I have send the error report again.

**Reply by devvaannsh** [reply]
Hey @arachnidus, apologies for not being able to fix this issue. Our development work for this release is already complete, so the fix can’t go into it, but I’ll aim to have it ready for the March release.

**Reply by arachnidus** [reply]
I hope so, I see that this is a common issue by GitHub developers and it is explained by Microsoft as a bug. 
As long as this is not fixed I can not switch to Phoenix and keep using Brackets

**Reply by devvaannsh** [reply]
@arachnidus apologies for the issues you are experiencing with Phoenix. I will work to have this fixed in the next release.

**Comment by arachnidus** [comment]
I have installed 5.1.3-22004 pre-release and the problem still exist in this pre-lease. Is it so difficult to fix this issue, maybe I can help
![Screenshot 2026-02-27 204950](https://github.com/user-attachments/assets/edfe6a73-3018-4ae1-8ed5-ca54f1c242e1)

**Reply by abose** [reply]
We were unable to reproduce this issue in any of our multi screen windows setups till now. If you have another windows pc, can you check if this issue is present in that. Itll help isolate if it is something device configuration specific

**Comment by arachnidus** [comment]
Hi Arun,

 

No, I don’t have another Windows PC. Only a laptop but the crash only happens when I move the window to another screen, not when I minimize/maximize.

 

I have three screens with an 8 Gb AMD Radeon RX6000. Each screen has a resolution of 2560 x 1440. The processor is a AMD Ryzen 7 5800X 8 core. I am using the AMD Software.

Can you prepare a Phoenix build with a debug logging and share it with me

**Comment by arachnidus** [comment]
Hi Arun,

 

Do you like to receive a user-mode dump file, full or mini to us ein WinDBG

**Reply by abose** [reply]
Try booting os is safe mode and safe mide with graphics enabled and see if the issue is still there. That will narrow it down to see if its any other optional drivers or installed apps interfering. We use tauri 1.5 so any tauri 1.5 apps should also fail likeley in that machine.

**Comment by beesey-sys** [comment]
Добрый день! Проблема осталась.  Выпуск	Windows 10 Домашняя
Версия	22H2
Дата установки	‎09.‎09.‎2020
Сборка ОС	19045.6456

<img width="308" height="167" alt="Безымянный" src="https://github.com/user-attachments/assets/98450704-a384-4f39-8336-cd5ed036dc6d" />

**Reply by abose** [reply]
Is it the same crash, on moving the window to a different screen? Can you give more info when did it crash - during startup or when moving the window to a different screen or something else.

**Comment by arachnidus** [comment]
In Safe mode even with graphics enabled only one screen is active.

**Comment by arachnidus** [comment]
I have good news, had to do a reinstall of my OS and problem is gone. Don't know what it, call is closed
